site/apache.md

```diff
@@ -1,476 +1,47 @@
-
-
-It is sometimes, but rarely, desirable to expose a WebLogic Administration Server host and port through a load balancer to a public network.  If this is needed, you can customize the exposure of the WebLogic Administration Server host and port by using the `loadBalancerExposeAdminPort` property in the `create-weblogic-domain-inputs.yaml` file.
-
-```
-
-# Boolean to indicate if the admin port is going to be exposed via APACHE load balancer. By default, it is false.
-loadBalancerExposeAdminPort: false
-```
-
-If the domain is created with the `loadBalancerExposeAdminPort` set to `true`, an adminstrator can access the Administration Console using `http://<host>:30305/console`.
-
-### Use your own plugin WL module configuration
-
-
-You can fine tune the behavior of the Apache plugin by providing your own Apache plugin configuration. You put your `custom_mod_wl_apache.conf` file in a local directory, for example, `<host-config-dir>` , and specify this location in the `create-weblogic-domain-inputs.yaml` file as follows:
-
-```
-
-# Docker volume path for APACHE
-
-# By default, the VolumePath is empty, which will cause the volume mount be disabled
-
-loadBalancerVolumePath: <host-config-dir>
-
-```
-
-After the `loadBalancerVolumePath` property is specified, the `create-weblogic-domain.sh` script will use the `custom_mod_wl_apache.conf` file in `<host-config-dir>` directory to replace what is in the Docker image.
-
-The generated YAML files will look similar except with un-commented entries like below:
-
-```
-
-      volumes:
-
-      - name: "domain1-apache-webtier"
-
-        hostPath:
-
-          path: <host-config-dir>
-
-      containers:
-
-      - name: domain1-apache-webtier
-
-        image: store/oracle/apache:12.2.1.3
-
-        imagePullPolicy: Never
+* Create your own `custom_mod_wl_apache.conf` file, and put it in a local directory, for example, `<host-conf-dir>`. See the instructions in [Apache Web Server with Oracle WebLogic Server Proxy Plugin on Docker](https://docs.oracle.com/middleware/1213/webtier/develop-plugin/apache.htm#PLGWL395).
 
-        volumeMounts:
+* Create a Kubernetes deployment to start an Apache load balancer instance. Note that you need to use the **volumes** and **volumeMounts** to mount `<host-config-dir>` into the `/config` directory inside the pod that runs the Apache web tier. Note that the Apache HTTP Server needs to be in the same Kubernetes namespace as the WebLogic domain that it needs to access.
 
-        - name: "domain1-apache-webtier"
+* Create a Kubernetes NodePort service to expose the internal ports to external access.
 
-          mountPath: "/config"
+* Create a RBAC YAML file to setup security for your load balancer. This is optional.
 
-```
+See samples for creating Apache HTTP Server as a load balancer at [Apache load balancer helm chart](../kubernetes/samples/charts/apache-webtier/README.md) and [Apache load balance samples](../kubernetes/samples/charts/apache-samples/README.md).
```

Comment:
See samples for creating Apache HTTP Server -> See the samples for creating the Apache HTTP Server